### DISCUSSION TITLE
LEVELING_AFTER_G28

### DISCUSSION THREAD

**Discussion by wbvczar** [mriscoc/Ender3V2S1#675] [Q&A] [answered]
I currently use the pre-compiled versions of this firmware for the 4.2.2 and 4.2.7 boards, manual mesh.  Today I was looking at the configuration.h file in the "Ender3V2-422-MM" folder, and I saw that both RESTORE_LEVELING_AFTER_G28 and ENABLE_LEVELING_AFTER_G28 were disabled in this file.

Are either of these options enabled in the pre-compiled versions?

**Comment by mriscoc** [comment, marked as answer]
No, enabling the leveling system automatically after G28 could have issues with tramming, Z offset and other features of the firmware, so, the leveling system is automatically disabled after G28, that is why a 'M420 S1' or 'G29 A' is required in the Slicer start gcode script.
